### ISSUE TITLE
[v4.4.0] Mist forgot to remove their profile

### ISSUE THREAD

**Issue by wolfieboy09** [Mistium/Origin-OS#33]
![image](https://github.com/Mistium/Origin-OS/assets/105564186/600120ca-b2a2-48a9-821e-b9ca67176142)
I loaded 4.4.0, and get this

**Comment by Mistium** [comment]
yep, ik, this is an issue with the actual reset drive system not that i forgot to remove my profile

**Comment by Mistium** [comment]
Should be fixed in v4.4.1
